src/views/accounts/AccountPush/AccountPushCreateUpdate.vue

```diff
@@ -42,7 +42,7 @@ export default {
           ]
         ],
         [
-          this.$t('Automations'), ['params']
+          this.$t('Params'), ['params']
         ],
         [this.$t('Periodic'), ['is_periodic', 'interval', 'crontab']],
         [this.$t('Other'), ['check_conn_after_change', 'is_active', 'comment']]
```

Comment:
The difference between the two codes lies in the titles of their associated methods (`Automations`, `Params`, and others). Both are identical; it's just the title formatting that differs. No other changes were made to optimize or verify anything about these lines of code since they're basic JavaScript imports.